tests.yaml

```diff
@@ -71,3 +71,42 @@ tests:
       exp_annotations:
         message: "Kubelet 'minikube' is running at 98.18% of its Pod capacity."
         runbook_url: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubelettoomanypods
+
+- interval: 1m
+  input_series:
+  - series: 'kube_pod_container_resource_requests_cpu_cores{container="kube-apiserver-67",endpoint="https-main",instance="10.131.2.78:8443",job="kube-state-metrics",namespace="kube-apiserver",node="ip-10-0-141-66.us-east-2.compute.internal",pod="kube-apiserver-ip-10-0-141-66.us-east-2.compute.internal",service="kube-state-metrics"}'
```

Comment:
can we add one more pod that is in "completed" state, just so we can make sure that the "exclusion" we want is truly done?